clippy_lints/src/if_then_some_else_none.rs

```diff
@@ -99,18 +100,26 @@ impl LateLintPass<'_> for IfThenSomeElseNone {
                     "consider using `bool::then` like: `{}.then(|| {})`",
                     cond_snip,
                     closure_body,
-                );
+                    );
                 span_lint_and_help(
                     cx,
                     IF_THEN_SOME_ELSE_NONE,
                     expr.span,
                     "this could be simplified with `bool::then`",
                     None,
                     &help,
-                );
+                    );
```

Comment:
rustfmt sadly doesn't work inside macros. Could you fix this indention and the one in line 103? Then it's all good :+1:

Reply:
Hm, I thought that this change was because of rustfmt 🤔

Reply:
These indentations also often happen in my editor, when I move code around. It's good that rustfmt is really solid otherwise, and this might be fixed when Rust supports `if let` chains in the future :upside_down_face: